.ci/clusters/values.yaml

```diff
@@ -120,6 +120,8 @@ broker:
     managedLedgerDefaultEnsembleSize: "1"
     managedLedgerDefaultWriteQuorum: "1"
     managedLedgerDefaultAckQuorum: "1"
+    enablePackagesManagement: "true"
```

Comment:
Does it make sense to just keep the following line?

Reply:
I copied it from [function-mesh-worker-service](https://github.com/streamnative/function-mesh-worker-service/blob/master/.ci/clusters/values_mesh_worker_service.yaml#L125-L126), @freeznet do you know what's `PULSAR_PREFIX_enablePackagesManagement` used for?

Reply:
please check https://github.com/apache/pulsar/blob/branch-2.10/docker/pulsar/scripts/apply-config-from-env.py, the `PULSAR_PREFIX_` is used to apply the ENV to config file, and `enablePackagesManagement` is to enable the package management service in the broker.